### PR TITLE
Fixing README updates

### PR DESCRIPTION
**Pull request by chrislovecnm**
I did not make the README change in the autogen directory.  Fixing that.

FIXES: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/190

### CODE REVIEW

**Comment by chrislovecnm** [comment]
```
       * google_container_node_pool.pools: Error reading instance group manager returned as an instance group URL: "Get https://www.googleapis.com/compute/beta/projects/ci-kubernetes-engine/zones/us-east4-b/instanceGroupManagers/gke-stub-domains-clu-default-node-poo-7844eead-grp?alt=json&prettyPrint=false: net/http: request canceled (Client.Timeout exceeded while awaiting headers)"
```

This is a new one.

**Comment by morgante** [comment]
Sounds like an underlying API timeout/issue, not anything in Terraform.

*Morgante Pell*

Strategic Cloud Engineer

morgantep@google.com

+1 212-565-7834

**Comment by chrislovecnm** [comment]
@aaron-lane / @morgante PTAL

File: autogen/README.md [hunk 1]
```diff
@@ -182,10 +182,9 @@ To more cleanly handle cases where desired functionality would require complex d
 
 The root module is generated by running `make generate`. Changes to this repository should be made in the [`autogen`](/autogen) directory where appropriate.
 
-Note: The correct sequence to update the repo using autogen
-functionality is the run `make generate && make generate_docs`.  This
-will create the various Terraform files, and then generate the
-Terraform documentation using `terraform-docs`.
+Note: The correct sequence to update the repo using autogen functionality is the run
```
Comment (morgante) [inline autogen/README.md]:
```suggestion
Note: The correct sequence to update the repo using autogen functionality is to run 
```

Comment (chrislovecnm) [reply autogen/README.md]: @aaron-lane just fixed it :)

Comment (morgante) [reply autogen/README.md]: Doesn't look fixed?

Comment (chrislovecnm) [reply autogen/README.md]: Now that I have not messed up git and rebasing ... https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/191/files#diff-34a5f0b5ab84cf5f33367f7912231babR185 

Now it is fixed

**Comment by chrislovecnm** [comment]
@morgante PTAL

**Pull request merged**